db/migrate/20230222155444_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorId
+      t.integer :postId
+      t.text :text
+      t.datetime :updatedAt
+      t.datetime :createdAt
```

Comment:
- [ ] Kindly delete these columns, these are not required because 👉🏽 `t.timestamps` will crate updated_at and created_at columns